sections/dashboard/DashboardCard/DashboardCard.tsx

```diff
@@ -42,11 +43,7 @@ const DashboardCard: FC = () => {
 	const router = useRouter();
 	const isL2 = useRecoilValue(isL2State);
 
-	const {
-		useExchangeRatesQuery,
-		useSynthsBalancesQuery,
-		useRedeemableDeprecatedSynthsQuery,
-	} = useSynthetixQueries();
+	const { useExchangeRatesQuery, useSynthsBalancesQuery } = useSynthetixQueries();
```

Comment:
I'm surprised `useSynthsBalancesQuery` doesn't make a lot of RPC calls

Reply:
Looks like we fetch the data using this: https://github.com/Synthetixio/js-monorepo/blob/d8b882a83e3682aee13c9c40f2174aa030097193/packages/queries/src/queries/walletBalances/useSynthsBalancesQuery.ts#L22

The contract call returns all the balances for the given `walletAddress`